test.py

```diff
@@ -97,13 +113,17 @@ def pretty_print(amr):
 print "\nAll the nodes in the amr should appear here.\n"
 print amr.keys()
 
-
 print "\nCreating custom AMR.\n"
 custom_AMR = CustomizedAMR()
 custom_AMR.create_custom_AMR(amr)
+print "\nCustom AMR token to concepts dict\n"
 print custom_AMR.tokens_to_concepts_dict
-print custom_AMR.relations_dict.keys()
-print custom_AMR.relations_dict
-
 
+parent_dict = {"l": "", "b": "l", "w": "b", "w2": "b", "a": "b"}
```

Comment:
@AndreeaLazar You can see here a mocked example of what the Customized AMR should look like